site/_posts/2021-07-03-avoid-lockin-insulate-your-application-and-have-backup-plan-excerpt.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: blog-post
+title: Avoid Lock-In by Insulating Your Application and Having a Backup Plan
```

Comment:
How to protect your organization from auth vendor lock-in

Reply:
Love it!